src/KeyboardInputModal.scss

```diff
@@ -18,6 +9,15 @@
     flex-direction: column;
     justify-content: center;
     margin-top: 1rem;
+
+    .modal {
+        border-radius: 0.25rem;
+    }
+    
+    .modal-body {
```

Comment:
same for the .modal-body, but we need padding:0 directly applied to .KeyboardInputModal__content, so updated KeyboardInputModal__content would be:

```
.keyboardInputModal__content {
display: flex;
flex-direction: column;
justify-content: center;
margin-top: 1rem;
padding: 0;
}
```